### ISSUE TITLE
FR doc: add markdownTooltip / disableToolTip example

### ISSUE THREAD

**Issue by GitMensch** [usernamehw/vscode-commands#44]
This is kind of a follow-up to #40 where it was implemented and API-documented in the README.
As the README has nice examples for (all?) other attributes: please add a doc entry, maybe the following would be a starting point

```md
## Tooltips

```js
"GitHub": {
    "icon": "github",
    "markdownTooltip": "## Description:\n\n- one\n- two",
},
"GitLab": {
    "icon": "gitlab",
    "disableToolTip": true,
},
```

**Comment by usernamehw** [comment]
Idk. I think README has too much stuff as it is. People are looking at it and go "Aww, hell naw":

![commands_installs](https://user-images.githubusercontent.com/9638156/220100884-86121a13-bc24-4b7e-a149-4e5f80e7e08b.png)

Maybe if extension had a dedicated docs page, it would fit there.

**Comment by GitMensch** [comment]
I agree that it would be reasonable to have a docs folder and move the samples there.
Something like https://github.com/redhat-developer/vscode-xml (linking to the docs page with all the details).
